libs/eo/metering-points/shell/src/lib/eo-metering-point-table.component.ts

```diff
@@ -135,7 +149,11 @@ import { EoBetaMessageComponent } from '@energinet-datahub/eo/shared/atomic-desi
       </ng-container>
 
       <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
-      <mat-row *matRowDef="let row; columns: displayedColumns"></mat-row>
+      <mat-row
+        *matRowDef="let row; columns: displayedColumns"
+        (mouseenter)="onRowHover(row.gsrn)"
```

Comment:
Suggestion: We should talk with UX about this since hover/mouse events don't work on touch devices.